repository/src/main/java/org/apache/atlas/repository/graph/IndexRecoveryService.java

```diff
@@ -248,6 +265,18 @@ private void stopMonitoring() {
         private void printIndexRecoveryStats() {
             this.graph.getManagementSystem().printIndexRecoveryStats(txRecoveryObject);
         }
+
+        private void releaseLock(InterProcessMutex lock) {
+            try {
+                if (isActiveActiveHAEnabled && lock != null && lock.isOwnedByCurrentThread()) {
+                    LOG.info("About to release index recovery lock");
+                    lock.release();
+                    LOG.info("successfully released index recovery lock");
+                }
+            } catch (Exception e) {
+                LOG.error(e.getMessage(), e);
```

Comment:
Let's improve all the error logs by adding a custom message, so we can search on sentry.

Reply:
This is addressed